go/exorelay/exorelay_test.go

```diff
@@ -19,13 +20,12 @@ func FeatureContext(s *godog.Suite) {
 	exocom := exocomMock.New()
 	go exocom.Listen(4100)
 
-	s.Step(`^an ExoRelay with the configuration:$`, func(configStr *gherkin.DocString) error {
-		var config map[string]interface{}
-		err := json.Unmarshal([]byte(configStr.Content), &config)
-		exoInstance = New("ws://localhost:4100", config)
-		if err != nil {
-			return err
-		}
+	s.Step(`^an ExoRelay with the role "([^"]*)"$`, func(role string) error {
+		exoInstance = New(ExoRelayConfig{
```

Comment:
I don't understand your comment. This is in the exorelay folder. I'm good adding tests for the exocom-mock (which I will expect we will need anyway and the javascript version has).